lib/routes/pollingRoutes.js

```diff
@@ -23,6 +24,26 @@ const createTerms = terms => (terms.active && terms.text) ? ({
   cancel: terms.cancelButtonText
 }) : null
 
+const buildTriggers = async (allTriggers) => {
+  const normalTriggers = []
+  const customTriggers = _.filter(o => {
+    if (o.customRequirementId === '') normalTriggers.push(o)
+    return o.customRequirementId !== ''
+  }, allTriggers)
+  const customTriggerDetails = await _.flow([_.map(_.get('customRequirementId')), customRequestQueries.batchGetCustomInfoRequest])(customTriggers)
```

Comment:
Is it safe to use `await` on l-s? I remember Rafael saying that on l-s it would be safer to continue using `then` due to something related to error-handling on a lower level (possibly the request client). I prefer `await` all the way, but it's something I remember being spoken about